src/leappto/providers/libvirt_provider.py

```diff
@@ -92,16 +93,23 @@ def __get_value(elem):
             else:
                 return ''
 
+        def __virt_inspector_supports_shallow():
+            """
+            :return: True if virt-inspector supports --no-applications and --no-icon
+            """
+            inspector_version = check_output(['virt-inspector', '-V'])
+            major, minor, _ = inspector_version.split(' ')[1].split('.', 2)
+            return int(minor) >= 34 or int(major) > 1
```

Comment:
Due to the way Python defines tuple comparison, this can be shortened to:

    return (int(major), int(minor)) >= (1, 34)